src/FieldThumbnailBrowser.cpp

```diff
@@ -122,6 +122,7 @@ void FieldBrowser::OnPaint(wxPaintEvent& event)
     auto offset_x = 0;
     auto offset_y = 0;
 
+    printf("%s\n", __PRETTY_FUNCTION__);
```

Comment:
Do you think we should keep this for future debugging?  Or was this just temporary from while you were working on this particular issue?

Reply:
Yeah I need to get rid of that...

Reply:
Done.